src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,94 @@
+// @import url("../utils/reset.css");
+// @import url("../utils/normalize.css");
+// @import url("../utils/variables.scss");
+// @import url("../utils/mixin.scss");
+
+$cell-border-size: 100px;
+$calendar-width: $cell-border-size*7 +10px;
+$gap: 1px;
+$days:
+  "mon" 0,
+  "tue" ($cell-border-size + $gap),
```

Comment:
Will be better to move this calculation to the place where you set corresponding property to the needed selector instead of repeating it several times